pkgs/development/python-modules/crypt4gh/default.nix

```diff
@@ -0,0 +1,55 @@
+{
+  lib,
+  buildPythonPackage,
+  setuptools,
+  fetchFromGitHub,
+  installShellFiles,
+  bats,
+  openssh,
+
+  # deps
+  cryptography,
+  bcrypt,
+  pyyaml,
+  docopt,
+  pynacl
+}:
+
+buildPythonPackage rec {
+  pname = "crypt4gh";
+  version = "1.7";
+  pyproject = true;
+  src = fetchFromGitHub {
+    owner = "EGA-archive";
+    repo = "crypt4gh";
+    rev = "v${version}";
+    hash = "sha256-kPXD/SityWscHuRn068E6fFxUjt67cC5VEe5o8wtxwk=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [ pyyaml docopt cryptography pynacl bcrypt ];
+
+  pythonImportsCheck = [ "crypt4gh" ];
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+      installShellCompletion --bash --name crypt4gh.bash $src/completions/crypt4gh.bash
+      installShellCompletion --bash --name crypt4gh-debug.bash $src/completions/crypt4gh-debug.bash
+      installShellCompletion --bash --name crypt4gh-keygen.bash $src/completions/crypt4gh-keygen.bash
+  '';
+
+  nativeCheckInputs = [ bats openssh ];
```

Comment:
Oh also, I think you should be using the optional `ssh` dependency of the cryptography package instead of adding openssh to the native inputs.

Reply:
Oh I see. That's fine then.